data/src/main/scala/ch.epfl.scala.index.data/bintray/DownloadPoms.scala

```diff
@@ -55,7 +55,7 @@ class DownloadPoms  (implicit val system: ActorSystem, implicit val materializer
   private val searchesBySha1: Set[BintraySearch] = {
 
     BintrayMeta.readQueriedPoms(bintrayCheckpoint)
-      .filter(s => !Files.exists(pomPath(s)) || !verifySHA1FileHash(pomPath(s), s.sha1))
+      .filter(s => !Files.exists(pomPath(s)))// || !verifySHA1FileHash(pomPath(s), s.sha1))
```

Comment:
@MasseGuillaume Doesn't seem to work, every time i get 53323 files to download, after commenting that out just 23.